src/main/scala/org/allenai/pipeline/Producer.scala

```diff
@@ -169,6 +168,22 @@ class PersistedProducer[T, -A <: Artifact](
   }
 
   override def stepInfo = step.stepInfo.copy(outputLocation = Some(artifact.url))
+
+  override def withCachingDisabled = {
```

Comment:
Fix bug in which disabling caching on a `PersistedProducer` turned it into a regular `Producer`